app/src/main/java/com/teamwss/websoso/ui/normalExplore/NormalExploreActivity.kt

```diff
@@ -0,0 +1,108 @@
+package com.teamwss.websoso.ui.normalExplore
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.WindowManager
+import androidx.activity.viewModels
+import com.teamwss.websoso.R
+import com.teamwss.websoso.databinding.ActivityNormalExploreBinding
+import com.teamwss.websoso.ui.common.base.BindingActivity
+import com.teamwss.websoso.ui.normalExplore.adapter.NormalExploreAdapter
+import com.teamwss.websoso.ui.normalExplore.model.NormalExploreUiState
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class NormalExploreActivity :
+    BindingActivity<ActivityNormalExploreBinding>(R.layout.activity_normal_explore) {
+    private val normalExploreAdapter: NormalExploreAdapter by lazy { NormalExploreAdapter(::navigateToNovelDetail) }
+    private val normalExploreViewModel: NormalExploreViewModel by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding.onClick = onNormalExploreButtonClick()
+        bindViewModel()
+        setupUI()
+        setupUiStateObserver()
+        setupSearchWordObserver()
+    }
+
+    private fun bindViewModel() {
+        binding.normalExploreViewModel = normalExploreViewModel
+        binding.lifecycleOwner = this
+    }
+
+    private fun onNormalExploreButtonClick() = object : NormalExploreClickListener {
+
+        override fun onBackButtonClick() {
+            finish()
```

Comment:
```suggestion
            override fun onBackButtonClick() = finish()
```
c: 이렇게 작성할 수도 있지 않을까요?

Reply:
약간 코드 스타일에 따라 다를 것 같은데 저는 = 보다 중괄호 사용을 더 선호하긴 해요! 
메서드들이 다 { } 로 통일성 있게 가져가는 것을 좋아해섭 ㅎ.ㅎ

Reply:
@yeonjeen 일반적으로 '='는 코틀린에서 반환형 함수를 나타냅니다!

저는 반환값이 없는 함수는 중괄호, 반환값이 있으며 한줄로 작성이 가능한 경우에 =을 활용합니다.